pkg/repository/chartmuseum.go

```diff
@@ -134,6 +132,10 @@ func (c *chartmuseum) Poll() {
 	}
 
 	updateRepo := false
```

Comment:
https://github.com/kubebb/core/pull/104/files?diff=split#diff-2e9abd09c0abdabe2a30a80ced0000f009dd3e909fc18ad52b113e1189e6affbR187-R191 var `updateRepo` and line 187-191 can also be deleted.